Apps/BRNPlayground/package.json

```diff
@@ -14,7 +14,8 @@
     "start": "react-native start",
     "test": "jest",
     "windows": "react-native run-windows --sln windows/BRNPlayground.sln",
-    "postinstall": "node ./postinstall.js"
+    "postinstall": "node ./postinstall.js",
+    "build-xcframeworks": "cd ../../Package && npx gulp createXCFrameworks"
```

Comment:
Should we run this as part of the postinstall? I think we would still want to be able to just run `npm install` in the directory and then be ready to build locally. If I'm understanding correctly, that would require an extra command as is?

Reply:
So the issue I saw with this is that even for Windows jobs we were building XCFrameworks which is less than ideal.. Not sure maybe we can scope it differenty.